mmf/modules/embeddings.py

```diff
@@ -322,22 +322,7 @@ def initialize_visual_from_pretrained(self):
             deepcopy(self.position_embeddings.weight.data), requires_grad=True
         )
 
-    def forward(
-        self,
-        input_ids,
-        token_type_ids=None,
-        visual_embeddings=None,
-        visual_embeddings_type=None,
-        position_embeddings_visual=None,
-        image_text_alignment=None,
-    ):
-        """
-        input_ids = [batch_size, sequence_length]
-        token_type_ids = [batch_size, sequence_length]
-        visual_embedding = [batch_size, image_feature_length, image_feature_dim]
-        image_text_alignment = [batch_size, image_feature_length, alignment_dim]
-        """
-
+    def encode_text(self, input_ids, token_type_ids=None):
```

Comment:
Can you add `*args, **kwargs` at the end to ensure easy forward compatibility?

Reply:
Thanks for the pointer. I have a qn about [where else it should go ](https://github.com/facebookresearch/mmf/pull/536/files/537fc81fd9a89c1f3b51fc020d55a2136d353b1b#r483316471)